src/components/ebay-notice/template.marko

```diff
@@ -3,7 +3,7 @@
 </script>
 
 <var type=(data.type || "page")/>
-<var status=(data.status || "priority")/>
+<var status=(data.status || "attention")/>
```

Comment:
Can we also check for `priority` for backwards compatibility? We can remove that alias in v3.

Reply:
So not sure if that is needed. The status is passed directly to skin to render that given priority. So passing priority will just pass it directly to skin. 
The only thing that was changed here was the default value.

Reply:
Ah, "priority" was the default? In that case you're right, it's not needed.